k-distribution/pl-tutorial/2_languages/1_simple/2_typed/1_static/simple-typed-static.md

```diff
@@ -1,13 +1,13 @@
 ---
-copyright: Copyright (c) K Team. All Rights Reserved.
+copyright: Copyright (c) Runtime Verification, Inc. All Rights Reserved.
 ---
 
 # SIMPLE — Typed — Static
 
-Author: Grigore Roșu (grosu@illinois.edu)  
+Author: Grigore Roșu (grosu@illinois.edu)
 Organization: University of Illinois at Urbana-Champaign
 
-Author: Traian Florin Șerbănuță (traian.serbanuta@unibuc.ro)  
+Author: Traian Florin Șerbănuță (traian.serbanuta@unibuc.ro)
```

Comment:
This is intentional. You can check how the rendering looks.
![Screenshot from 2023-12-15 18-18-13](https://github.com/runtimeverification/k/assets/5113255/62249c78-741c-435b-aa05-91f57afc91f5)
![Screenshot from 2023-12-15 18-18-32](https://github.com/runtimeverification/k/assets/5113255/c1855238-73f0-4ea6-b945-c40c6e58b089)